app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor["name"]}"
+                                     f" is not vaccinated.")

```

Comment:
The f-string syntax is incorrect here. You should use single quotes around the dictionary key `name` to avoid syntax errors. Change `f"{visitor["name"]}"` to `f"{visitor['name']}"`.